views/player_options.erb

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
As there is no conditional logic in this view, I didn't cover it with an erb test. The other one, which contains logic, is covered however.